### PR TITLE
Update ICSE 2022

### PR DESCRIPTION
**Pull request by Antimony5292**
### Which conference does this PR update?

- ICSE 2022

### Related URL

- https://conf.researchr.org/home/icse-2022

### CODE REVIEW

**Comment by jacklightChen** [comment]
@Antimony5292, we are sorry for the modifications during your pr, please sync the latest head~

**Comment by Antimony5292** [comment]
> @Antimony5292, we are sorry for the modifications during your pr, please sync the latest head~

Done :)

File: conference/SE/icse.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+- title: ICSE
+  description: 44th International Conference on Software Engineering
```
Comment (jacklightChen) [inline conference/SE/icse.yml:2]: now we can omit the `44th`

File: conference/SE/icse.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  dblp: icse
+  confs:
+    - year: 2022
+      id: icse22
+      link: https://conf.researchr.org/home/icse-2022
+      timeline:
+        - abstract_deadline: '2021-08-27 23:59:59'
+          deadline: '2021-9-3 23:59:59'
+      timezone: AoE
+      date: June 23-25, 2020
```
Comment (jacklightChen) [inline conference/SE/icse.yml:14]: the date may be wrong?

File: conference/SE/icse.yml [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  sub: SE
+  rank: A
+  dblp: icse
+  confs:
+    - year: 2022
+      id: icse22
+      link: https://conf.researchr.org/home/icse-2022
+      timeline:
+        - abstract_deadline: '2021-08-27 23:59:59'
+          deadline: '2021-9-3 23:59:59'
```
Comment (jacklightChen) [inline conference/SE/icse.yml:12]: format like '2021-09-03 23:59:59' is recommended~

**Review by jacklightChen — APPROVED** [review]

**Pull request merged**